keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/kep.yaml

```diff
@@ -22,12 +22,12 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.30"
+latest-milestone: "v1.31"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.30"
-  beta: "v1.31"
+  alpha: "v1.31"
+  beta: "v1.32"
```

Comment:
As no PRR review or KEP update happened before v1.32 enhancements freeze, this is missed.

```
-  alpha: "v1.31"
```
The alpha update is valid. beta may be 1.33 or later.